cves/kernel/CVE-2020-11494.yml

```diff
@@ -173,8 +170,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: The tool need to be aware of a specific struct and CAN subsystem
```

Comment:
Probably explain this either here or in the vulnerability description since I don't think many would know what a CAN subsystem is.